pkgs/desktops/cinnamon/cinnamon-session/default.nix

```diff
@@ -71,6 +73,7 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [
     meson
     ninja
+    cmake
```

Comment:
Is CMake really needed or another instance of https://github.com/jtojnar/nixpkgs-hammering/blob/a24b5537fab53dec6dc638576bff19606ad4a33b/explanations/meson-cmake.md?

Reply:
it seems like it's not explicitly necessary, so I'll remove that